sdk/storage/azblob/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+* Fixed a bug to ensure crc64 response header is returned in Put Blob request. Fixes [#23372](https://github.com/Azure/azure-sdk-for-go/issues/23372).
```

Comment:
Technically this should be listed as a feature (new public surface area is always a feature).

Reply:
Moved to Features list